reports/historical/_gmmp_2015_importer.py

```diff
@@ -304,6 +317,18 @@ def import_6(self, sheet_data):
             row_heading_col=2,
         )
 
+        for year, country_data_per_media_per_region in all_data.items():
+            if year != 2010:
+                # 2015 data introduced the media rows.
+                country_data_per_media_per_region = {**country_data_per_media_per_region['print, radio, television'],
+                                                        **country_data_per_media_per_region['internet, twitter']}
+            year_data = {}
+            for continent, continent_data in country_data_per_media_per_region.items():
```

Comment:
```suggestion
            for region, region_data in country_data_per_media_per_region.items():
```
Or why the switch to `continent`?